SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorCommandLineArgsManager.cpp

```diff
@@ -0,0 +1,305 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialGDKEditorCommandLineArgsManager.h"
+
+#include "Input/Reply.h"
+#include "IOSRuntimeSettings.h"
+#include "Logging/LogMacros.h"
+#include "Misc/FileHelper.h"
+#include "Misc/MessageDialog.h"
+#include "Serialization/JsonSerializer.h"
+#include "SpatialCommandUtils.h"
+#include "SpatialGDKSettings.h"
+#include "SpatialGDKEditorSettings.h"
+
+#ifdef ENABLE_LAUNCHER_DELEGATE
+//need this macro before "ILauncherServicesModule.h" to prevent compile error
+#define LAUNCHERSERVICES_API
+
+#include "Developer\LauncherServices\Public\ILauncherServicesModule.h"
+#include "Developer\LauncherServices\Public\ILauncherWorker.h"
+#include "Developer\LauncherServices\Public\ILauncher.h"
+#endif
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialGDKEditorCommandLineArgsManager, Log, All);
+DEFINE_LOG_CATEGORY(LogSpatialGDKEditorCommandLineArgsManager);
+
+FSpatialGDKEditorCommandLineArgsManager::FSpatialGDKEditorCommandLineArgsManager()
+#ifdef ENABLE_LAUNCHER_DELEGATE
+	: bAndroidDevice(false)
+	, bIOSDevice(false)
```

Comment:
I think we can remove `bIOSDevice`. It doesn't look like we are using it anymore